charts/tekton-apps/README.md

```diff
@@ -342,6 +351,12 @@ spec:
                   pm: xxx
                   tm: xxx
                 namespace: prod
+                notifications:
```

Comment:
Doesn't it make sense to move this to `tektons-apps` helmchart and leave here only something like 
```yaml
notifications: true
```

Reply:
It's also an option.

Assuming that the slack channels are always the project name, we could put in `charts/tekton-apps/templates/argocd_projects.yaml `

```
    notifications.argoproj.io/subscribe.on-health-degraded.slack: project-{{ $project.project }}; project-{{ $project.project }}-alarms
    notifications.argoproj.io/subscribe.on-sync-failed.slack: project-{{ $project.project }}-ci; project-{{ $project.project }}-alarms
    notifications.argoproj.io/subscribe.on-sync-status-unknown.slack: project-{{ $project.project }}; project-{{ $project.project }}-alarms
    notifications.argoproj.io/subscribe.on-sync-succeeded.slack: project-{{ $project.project }}-ci; project-{{ $project.project }}-alarms
```

Reply:
I like that notifications are verbose. `notifications: true` is very limited